source/User_Guide/Settings/two_factor_authentication.md

```diff
@@ -29,94 +29,50 @@ Setting Up Two-Factor Authentication
 You may choose from two different verification methods when using two-factor authentication:
```

Comment:
the beginning of this page has a the stutter

SendGrid’s Two-Factor Authentication can be configured to accept a code sent to your mobile phone either via the the Authy App, or an SMS message